pom.xml

```diff
@@ -16,16 +16,11 @@
   <name>guacamole: variant caller</name>
 
   <properties>
-    <adam.version>0.19.0</adam.version>
```

Comment:
I'd agree that it's not worth it to factor out for a single dependency, but once there seems useful, easy to get a read on the major dependencies and their version ... but probably fine as is.